docs/csv.md

```diff
@@ -0,0 +1,115 @@
+# CSV import
+
+`reccmp` has the option to parse [comma-separated value (CSV)](https://en.wikipedia.org/wiki/Comma-separated_values) files and load data to supplement code annotations on the original binary.
+
+Any metadata set from a CSV file will overwrite the same data from code annotations on the same address.
+
+If the same address is repeated on multiple lines the same CSV file, or across multiple files, the metadata will be overwritten each time.
+
+## Setup
+
+Add the list of CSV files under the `data-sources` key for the target in `reccmp-project.yml`. For example:
+
+```yml
+targets:
+  ISLE:
+    filename: ISLE.EXE
+    source-root: .
+    hash:
+      sha256: 5cf57c284973fce9d14f5677a2e4435fd989c5e938970764d00c8932ed5128ca
+    data-sources:
+      - test.csv
+```
+
+The paths are relative to the directory that contains `reccmp-project.yml`.
+
+## Format
+
+Here's an example of a CSV file using the pipe character[^1] as the delimiter.
+
+```
+address|name|size
+1008b400|_atol|164
+1008b4b0|_atoi|14
+1008b4c0|_strtok|216
+1008b5a0|_sprintf|103
+1008b608|__ftol|39
+```
+
+The header on the first line defines the columns used in the file. The only required column is `"address"`. You can use the abbreviated `"addr"` if you prefer, but not both in the same file. `"address"` does not need to appear first, but this is common.
```

Comment:
As discussed in the other threads, I'd remove `addr` in favour of `address` both in code an in the docs